tests/test_resources.py

```diff
@@ -48,6 +48,7 @@ def test_external(self):
         )
         app.layout = dcc.Markdown()
         app.scripts.config.serve_locally = False
+        app.scripts.config.xxx = 123
```

Comment:
Hun?

Reply:
Senator, I do not recall.
https://github.com/plotly/dash/pull/722/commits/c096aa1c445dada79ef4d81c146328253a742390

Reply:
I saw it at the time and had assumed it was to test for acceptance of randomly-named config keys :)